query/src/main/java/org/infinispan/search/mapper/mapping/impl/InfinispanIndexedTypeContextImpl.java

```diff
@@ -83,6 +88,16 @@ public void dirtyFilter(PojoPathFilter dirtyFilter) {
          // Nothing to do
       }
 
+      @Override
+      public void applyLoadingBinder(Object binder, PojoEntityLoadingBindingContext context) {
+         var castBinder = (InfinispanSelectionLoadingBinder) binder;
+         this.loadingStrategy = castBinder.createLoadingStrategy(??? entityLoader ???);
+         if ( this.loadingStrategy != null ) {
+            context.selectionLoadingStrategy( typeIdentifier.javaClass(), this.loadingStrategy );
+//            context.massLoadingStrategy( typeIdentifier.javaClass(), this.loadingStrategy );
```

Comment:
is this needed ?

Reply:
No. It could be used, eventually, but currently Infinispan implements mass indexing without relying on this.